### PR TITLE
Added MPPositionProvider examples with ciscodna,gps and indooratlas. …

### PR DESCRIPTION
**Pull request by timMikkelsen**
…Also added a reverse geocode example

### CODE REVIEW

File: MapsIndoorsSamples/app/src/main/java/com/mapspeople/mapsindoorssamples/ui/positioning/CiscoDNA/CiscoDNAEntry.kt [hunk 1]
```diff
@@ -0,0 +1,134 @@
+package com.mapspeople.mapsindoorssamples.ui.positioning.CiscoDNA
```
Comment (matiasholst) [inline MapsIndoorsSamples/app/src/main/java/com/mapspeople/mapsindoorssamples/ui/positioning/CiscoDNA/CiscoDNAEntry.kt:1]: Is this class necessary? It is already implemented in the SDK

File: MapsIndoorsSamples/app/src/main/java/com/mapspeople/mapsindoorssamples/ui/reverseGeoCode/ReverseGeoCodeFragment.kt [hunk 2]
```diff
@@ -0,0 +1,102 @@
+package com.mapspeople.mapsindoorssamples.ui.reverseGeoCode
+
+import android.annotation.SuppressLint
+import android.os.Bundle
+import android.util.Log
```
Comment (matiasholst) [inline MapsIndoorsSamples/app/src/main/java/com/mapspeople/mapsindoorssamples/ui/reverseGeoCode/ReverseGeoCodeFragment.kt:5]: I do not know about others, but this is an unused import

File: MapsIndoorsSamples/app/src/main/java/com/mapspeople/mapsindoorssamples/ui/reverseGeoCode/ReverseGeoCodeFragment.kt [hunk 3]
```diff
@@ -0,0 +1,102 @@
...
+
+            setupListener()
+        }
+    }
+
+    @SuppressLint("SetTextI18n")
+    private fun setupListener() {
+        mMapControl?.let { mapControl ->
+            //Use a MapClick Listener to get a coordinate to perform the reverse geocode with.
+            mapControl.setOnMapClickListener { mpLatLng, mpLocations ->
```
Comment (matiasholst) [inline MapsIndoorsSamples/app/src/main/java/com/mapspeople/mapsindoorssamples/ui/reverseGeoCode/ReverseGeoCodeFragment.kt:86]: If you want to explicitly ignore the locations returned, replace the `mpLocations` with `_`

**Review by matiasholst — CHANGES_REQUESTED** [review]
Minor changes from other comments

File: MapsIndoorsSamples/app/src/main/java/com/mapspeople/mapsindoorssamples/ui/reverseGeoCode/ReverseGeoCodeFragment.kt (hunk 3 above)
Comment (timMikkelsen) [reply MapsIndoorsSamples/app/src/main/java/com/mapspeople/mapsindoorssamples/ui/reverseGeoCode/ReverseGeoCodeFragment.kt:86]: As this is for documentation purpose, i like that the users know what the listener returns (espescially in the code examples), and is it is referenced in the comment below i think it is best to keep it commented, even if our linter disapproves

**Review by matiasholst — APPROVED** [review]

**Pull request merged**